src/arviz_stats/bayes_factor.py

```diff
@@ -43,11 +43,11 @@ def bayes_factor(idata, var_name, ref_val=0, return_ref_vals=False, prior=None):
            ...: from arviz_base import from_dict
            ...: import arviz_stats as azs
            ...: idata = from_dict({"posterior":{"a":np.random.normal(1, 0.5, (2, 1000))},
-           ...:                    {"prior":{"a":np.random.normal(0, 1, (2, 1000))}}})
+           ...:                    "prior":{"a":np.random.normal(0, 1, (2, 1000))}})
```

Comment:
The example should be 

import numpy as np
from arviz_base import from_dict
import arviz_stats as azs
dt = from_dict({"posterior":{"a":np.random.normal(1, 0.5, (2, 1000))},
                    "prior":{"a":np.random.normal(0, 1, (2, 1000))}})
azs.bayes_factor(dt, var_name="a", ref_val=0)